corehq/apps/locations/templates/locations/manage/location.html

```diff
@@ -156,65 +156,75 @@
 
       <div class="tab-content">
         <div class="tab-pane {% if form_tab == "basic" %}active{% endif %}" id="basic-info">
-          <form id="loc_form" class="form form-horizontal" name="product" method="post">
-            {% bootstrap_form_errors form %}
-            {% crispy form.location_form %}
-
-            {% crispy form.custom_location_data.form %}
-
-            {% if form.user_form %}
-            <!-- ko if: has_user -->
-            {% crispy form.user_form %}
-            {% crispy form.custom_user_data.form %}
-            <!-- /ko -->
-            {% endif %}
-
-            <input type="hidden" name="form_type" value="location-settings" />
-
-            {% if consumption %}
-              <legend>{% trans "Default monthly consumption values" %}</legend>
-              {% for code, amount in consumption %}
-                <div class="form-group">
-                  <label class="col-sm-3 col-md-4 col-lg-2 control-label">{{ code }}</label>
-                  <div class="col-sm-4 col-md-5 col-lg-3 controls">
-                    <span class="form-control uneditable-input">
-                      {{ amount }}
-                    </span>
+
```

Comment:
I recommend ignoring whitespace for this commit: https://github.com/dimagi/commcare-hq/pull/17580/commits/ffd4034dbbf040f19498b788554e11e0c54773ba?w=1